website/blog/2022-06-16-resources-migrating-your-react-native-library-to-the-new-architecture.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Resources - Migrating Your React Native Library to the New Architecture
+author: 'cipolleschi'
+tags: [announcement]
+date: 2022-06-16
+---
+
+**tl; dr**: We are working on improving the resources supporting the react Native New Architecture. We already released a repository to help migrating your app ([RNNewArchitectureApp](https://github.com/react-native-community/RNNewArchitectureApp)) and one for your libraries ([RNNewArchitectureLibraries](https://github.com/react-native-community/RNNewArchitectureLibraries)). We are also revamping the [New Architecture guide](https://github.com/facebook/react-native-website/pull/3037) on the Website and we created a [GitHub Working Group](https://github.com/reactwg/react-native-new-architecture/discussions) to answer questions related to the new Architecture.
+
+## Introduction
+
+In this post we are sharing an update on tools and resources to help React Native library developers to migrate their **Native Modules** and **Native Components** to their **New Architecture** equivalents, **TurboModule** and **Fabric Components**.
+
+React Native users leverage a vast number of open-source libraries for their apps. For a complete and consistent ecosystem, it is necessary that these libraries migrate such that everyone can benefit from the unlocked capabilities and performance improvements of the new architecture.
+
+Here is what we’re working on to support library developers to migrate to the new architecture:
+
+- **Documentation:** We are expanding the [New Architecture guide](https://github.com/facebook/react-native-website/pull/3037) on the website to cover more concepts of the new architectures and how to develop your components.
+- **Example Migrations:** We’ve set up two repositories to demonstrate how to migrate a React Native app to the New Architecture ([RNNewArchitectureApp](https://github.com/react-native-community/RNNewArchitectureApp)) and how to create a **Fabric Component** and a **TurboModule** that work with both architectures ([RNNewArchitectureLibraries](https://github.com/react-native-community/RNNewArchitectureLibraries)).
+- **Support:** Earlier this year, we created a [GitHub Working Group](https://github.com/reactwg/react-native-new-architecture/discussions) dedicated to discussion and questions around the New Architecture.
+
+In this post, we will dig deeper into these resources and explain in more detail how you can use them most efficiently. Finally, we will provide a snapshot of the current migration state for the most used React Native libraries.
+
+### Documentation
+
+In the past 6 months, we’ve added a guide on adopting the New Architecture (https://reactnative.dev/docs/new-architecture-intro) and an architecture deep-dive (https://reactnative.dev/architecture/overview) on Fabric. We plan to expand this to include more guides and documentation around creating TurboModules, understanding CodeGen, and more. We hope to have updates to share by the 0.70 release.
```

Comment:
```suggestion
In the past 6 months, we’ve added a guide on adopting the New Architecture (/docs/new-architecture-intro) and an architecture deep-dive (/architecture/overview) on Fabric. We plan to expand this to include more guides and documentation around creating TurboModules, understanding CodeGen, and more. We hope to have updates to share by the 0.70 release.
```

Let's use internal links for doc pages. Those prevents the whole app rerender on navigation which lead to more seamless and quicker page loads.

Reply:
It also would be nice to add links to the certain words instead of listing the whole URL in content.
<img width="1047" alt="Screenshot 2022-06-17 at 09 24 04" src="https://user-images.githubusercontent.com/719641/174247433-15ce6eb8-3095-400f-a518-4deeb0f2d688.png">

Reply:
Thanks for the explanation about why it is useful to use internal links!
I missed these two spots for use words for links! 😱